pkg/migration/register.go

```diff
@@ -0,0 +1,7 @@
+package migration
+
+func init() {
```

Comment:
Add safeguard comment to not mess around with existing migrations.

Reply:
done